hopsworks-common/src/main/java/io/hops/hopsworks/common/featurestore/utils/FeaturestoreInputValidation.java

```diff
@@ -0,0 +1,85 @@
+package io.hops.hopsworks.common.featurestore.utils;
```

Comment:
You are missing the copyright header here

Reply:
added